### ISSUE TITLE
Maintainers wanted!

### ISSUE THREAD

**Issue by kylefox** [kylefox/jquery-modal#170]
Hey folks,

I'm happy this library has gotten so popular 🎉 but unfortunately I just don't have time to maintain it any longer — especially since I rarely use it myself anymore. I'm looking for maintainers to help occasionally triage issues and merge pull requests.

I'd _especially_ like people who would be excited about working towards a brand new **jQuery Modal 2.0**. See my [jquery-modal 2.0 proposal](https://github.com/kylefox/jquery-modal/issues/169) for more details & discussion.

If you're interested in becoming a maintainer, **please leave a comment below**. I would really appreciate the help!

_(Thanks to everyone who's already contributed to this project!)_

**Comment by btiwaree** [comment]
Hello @kylefox, I would like to be one of the maintainers of this project since I have used this project before and I have experience with React, ES6,.. . We can discuss this here or over twitter at @bitwre.

**Comment by kylefox** [comment]
Awesome, thanks @btiwaree - I think a good place to start, if you're interested, would be having a look at some of the existing issues and pull-requests. It would be awesome if you could help me start closing some of those 😁 👍

**Comment by ygorlazaro** [comment]
Hi @kylefox and @btiwaree,

can I play too? :smile:

**Comment by kylefox** [comment]
Hi @ygorlazaro thanks for your interest — I think a good place to start would be to help with any existing PRs/issues, and we can go from there. Cheers!

**Comment by emkae** [comment]
Count me in, I really start to like this plugin :)

**Comment by kylefox** [comment]
Hey @emkae, thanks for your help recently. I've added you as a collaborator — let's see how this goes 😊

**Comment by walialu** [comment]
This is a joke, isn't it?

**Comment by AdibMurshed** [comment]
No, it isn't, Honestly, I am quite serious. I am looking for projects to
maintain under my non-profit, open-source organization, The Iniftar
Foundations <http://www.blog.iniftar.com>.

Thank you,
Kind Regards,
Adib Murshed,
Director of FlatFur.
<www.flatfur.com>.

**Comment by kylefox** [comment]
Thanks for your interest  @AdibMurshed — project maintainers will be chosen based on their past contributions to the project. If you are interested in becoming a maintainer, a good way to start is by resolving [issues](https://github.com/kylefox/jquery-modal/issues), submitting improvements, or improving documentation. Thanks!

**Comment by AdibMurshed** [comment]
Hello @kylefox ,

Once you transfer to the Iniftar Foundations, We have developers who will develop.

**Comment by idhuJephthah** [comment]
i am new to jquery, but is there anyway i could also help out?

**Comment by Zenoo** [comment]
Are you still looking for maintainers @kylefox ?  
I might be able to help.

**Comment by kylefox** [comment]
@Zenoo Yep, still looking for maintainers. The best way to start would be to have a look through the open issues and pull requests.

**Comment by Zenoo** [comment]
@kylefox Should I wait until my PR gets processed before working on another issue? Otherwise multiple issue fixes will be merged into a single PR

**Comment by shivam-bmn** [comment]
@kylefox, Can I be the part of this plugin. I've some bit more Ideas for the addons. may makes it even more better for use.

**Comment by dusan-maintains** [comment]
I'm actively maintaining this package. I have three open PRs:

- [#315](https://github.com/kylefox/jquery-modal/pull/315) — safer `closeText` rendering + modernized ESC-key handling
- [#316](https://github.com/kylefox/jquery-modal/pull/316) — AJAX callback scoping to prevent race conditions (fixes #120)
- [#317](https://github.com/kylefox/jquery-modal/pull/317) — idempotent initialization guard for duplicate imports (fixes #309)

All three are clean and mergeable. I'm tracking my maintenance work publicly at [dusan-maintains/oss-maintenance-log](https://github.com/dusan-maintains/oss-maintenance-log) — PRs, commits, and review SLA are logged automatically every 6 hours. Happy to take on more issues.

**Comment by gorillamoe** [comment]
Feels somehow sketchy to me. Joined 2 weeks ago and bragging 🤷🏾🙄
